src/components/Exchange/PositionSeller.js

```diff
@@ -725,9 +727,11 @@ export default function PositionSeller(props) {
       withdrawETH, // _withdrawETH
     ];
 
-    const successMsg = t`Requested decrease of ${position.indexToken.symbol} ${
-      position.isLong ? "Long" : "Short"
-    } by ${formatAmount(sizeDelta, USD_DECIMALS, 2)} USD.`;
+    const successMsg = t`Requested decrease of ${position.indexToken.symbol} ${longOrShortText} by ${formatAmount(
+      sizeDelta,
+      USD_DECIMALS,
+      2
+    )} USD.`;
```

Comment:
Should we add a non breaking space here? 
And, in other places like {amount} {symbol} when it can cause unexpected line breaks

Reply:
Not critial just if you know some 'danger' places